### ISSUE TITLE
index.js files not being created and causing  BadRequestError: Bad Request and 500 errors

### ISSUE THREAD

**Issue by Paddy-Hamilton** [zeit/next.js#10825]
# Bug report

## Describe the bug

After creating a new `create-next-app` build and running yarn dev it looks like everything is running fine, and I get 
`[ wait ]  starting the development server ...`
`[ info ]  waiting on http://localhost:3000 ...`
`[ ready ] compiled successfully - ready on http://localhost:3000`

Then when I go to `http://localhost:3000` I get a blank white screen and loads of 500 errors
![Screenshot 2020-03-04 at 09 19 17](https://user-images.githubusercontent.com/9196768/75864040-54d97a00-5df9-11ea-81de-7a4a5c28030d.png)

and lots of these in the terminal 

`
BadRequestError: Bad Request
    at SendStream.error (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/send/index.js:270:31)
    at SendStream.pipe (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/send/index.js:520:10)
    at /Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/next-server/server/serve-static.js:17:14
    at new Promise (<anonymous>)
    at Object.serveStatic (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/next-server/server/serve-static.js:8:12)
    at DevServer.serveStatic (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/next-server/server/next-server.js:786:34)
    at DevServer.servePublic (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/server/next-dev-server.js:17:13)
    at DevServer._beforeCatchAllRender (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/server/next-dev-server.js:8:311)
    at async Object.fn (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/server/next-dev-server.js:12:4)
    at async Router.execute (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/next-server/server/router.js:117:32)
    at async DevServer.run (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/next-server/server/next-server.js:475:29)
    at async DevServer.handleRequest (/Users/paddyhamilton/Documents/sites/10%/nextstarter/node_modules/next/dist/next-server/server/next-server.js:131:20) {
  message: 'Bad Request',
  expose: false
}
`

**It looks like the index.js static file is not being generated as it doesn't exist in the `.next` folder in either the `server/static/development/pages` folder or the `static/development/pages` folder**
 
## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. follow the official nextjs Getting Started tutorial using `create-next-app` https://nextjs.org/docs/getting-started
2. yarn dev
3. go to http://localhost:3000

## Expected behavior

The tutorial index page should load

## System information

- OS: macOS
- Browser: chrome
- Version of Next.js:  9.2.2

**Comment by Paddy-Hamilton** [comment]
It seems this error is caused by the files project being located in a folder called `10%` and so in the `next-server/sever/serve-static.js` this passes the path to `send` which tries to decode the path of the pages in the .next project folder and the % causes an error in this decoding.  

Have fixed by giving my project folder a friendlier name for decoding.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
